examples/abitest/module_0/rust/src/lib.rs

```diff
@@ -228,6 +228,10 @@ impl OakAbiTestService for FrontendNode {
             (Self::test_channel_close_raw, Count::Unchanged),
         );
         tests.insert("ChannelClose", (Self::test_channel_close, Count::Unchanged));
+        tests.insert(
+            "HandleClone",
```

Comment:
nit: tests that mess with raw pointers etc normally have a "Raw" suffix in their names.